### ISSUE TITLE
Bug: Sqlite: Guids not working in where query 

### ISSUE THREAD

**Issue by Swoorup** [mikependon/RepoDB#593]
From gitter: 

The following query silently fails to give any results:

```fsharp
let ids: Guid list = ...
conn.Query((fun (e:EventReadEntity) -> ids.Contains(e.id)), orderByPositionAsc)
```

whereas the following gives result:

```fsharp
let ids: Guid list = ...
conn.ExecuteQuery<EventReadEntity>("SELECT * FROM es_events WHERE id in (@ids)", {|ids = ids.Select(fun i -> i.ToString())|})
```

In both cases, the same guids where passed against the same table and data. It appears some issue with the DbConversion.

**Comment by mikependon** [comment]
This is now available at [RepoDb.SqLite v1.1.0](https://www.nuget.org/packages/RepoDb.SqLite/1.1.0).

**Comment by Swoorup** [comment]
Can confirm this is resolved in v1.1.0 @mikependon
